### PR TITLE
Boilerplate for likert question

### PR DESCRIPTION
**Pull request by relumesaros**

### CODE REVIEW

**Comment by lakatosandrei** [comment]
Nu dam merge

File: packages/likert/configure/src/main.jsx [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    prompt = {},
+    teacherInstructions = {},
+    studentInstructions = {},
+    scoringType = {},
+  } = configuration || {};
+  const {
+    teacherInstructionsEnabled
+  } = model || {};
+
+  const valuesMap = model.choices.reduce((acc, choice) => {
```
Comment (elodszopos) [inline packages/likert/configure/src/main.jsx:206]: This needs to be moved outside and made into a reusable utility function.

File: packages/likert/configure/src/index.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    if (this._model) {
+      this.onModelChanged(this._model);
+    }
+
+    this._render();
+  }
+
+  /** @param {done, progress, file} handler */
+  insertImage(handler) {
```
Comment (elodszopos) [inline packages/likert/configure/src/index.js:86]: Image handling and support is most likely not needed for Likert.

File: packages/likert/configure/src/main.jsx (hunk 1 above)
Comment (relumesaros) [reply packages/likert/configure/src/main.jsx]: solved

File: packages/likert/configure/src/index.js (hunk 2 above)
Comment (relumesaros) [reply packages/likert/configure/src/index.js]: solved

**Comment by relumesaros** [comment]
Hi @edeustace 
Can we please also merge this merge ? Or review what changes need to occur in order to be approved?

File: packages/likert/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "access": "public"
+  },
+  "repository": "pie-framework/pie-elements",
+  "version": "0.0.1",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "@pie-ui/likert": "^0.0.1"
```
Comment (edeustace) [inline packages/likert/package.json:13]: hi @relumesaros - can you bump the dependency here? to `^0.1.2` - i'll then merge - thanks

Comment (relumesaros) [reply packages/likert/package.json]: solved :)

Comment (relumesaros) [reply packages/likert/package.json]: @edeustace this is gonna be published automatically? :)

Comment (edeustace) [reply packages/likert/package.json]: once I merge to master yes - but I'll need to coordinate that

**Pull request merged**